test/size.txt

```diff
@@ -1,55 +1,55 @@
   94.1 kB   |     7.16 kB   |        3 kB   |   v0/amp-jwplayer-0.1.js
-128.17 kB   |     8.02 kB   |     3.31 kB   |   v0/amp-kaltura-player-0.1.js
-148.44 kB   |    11.73 kB   |     4.42 kB   |   v0/amp-lightbox-0.1.js
- 96.79 kB   |     5.66 kB   |     2.46 kB   |   v0/amp-list-0.1.js
-166.09 kB   |    13.17 kB   |     5.01 kB   |   v0/amp-live-list-0.1.js
-150.31 kB   |    39.73 kB   |    14.28 kB   |   v0/amp-mustache-0.1.js
- 87.25 kB   |      6.5 kB   |     2.75 kB   |   v0/amp-o2-player-0.1.js
-138.55 kB   |    20.11 kB   |     6.03 kB   |   v0/amp-pinterest-0.1.js
+128.32 kB   |     8.02 kB   |     3.31 kB   |   v0/amp-kaltura-player-0.1.js
+148.43 kB   |    11.71 kB   |     4.42 kB   |   v0/amp-lightbox-0.1.js
+ 96.94 kB   |     5.66 kB   |     2.46 kB   |   v0/amp-list-0.1.js
+166.22 kB   |    13.13 kB   |        5 kB   |   v0/amp-live-list-0.1.js
+150.46 kB   |    39.73 kB   |    14.29 kB   |   v0/amp-mustache-0.1.js
+ 87.25 kB   |      6.5 kB   |     2.76 kB   |   v0/amp-o2-player-0.1.js
+ 138.7 kB   |    20.11 kB   |     6.03 kB   |   v0/amp-pinterest-0.1.js
  86.24 kB   |     5.99 kB   |     2.55 kB   |   v0/amp-reach-player-0.1.js
- 89.48 kB   |     5.33 kB   |     2.32 kB   |   v0/amp-share-tracking-0.1.js
+ 89.49 kB   |     5.33 kB   |     2.32 kB   |   v0/amp-share-tracking-0.1.js
  99.81 kB   |     9.88 kB   |     3.73 kB   |   v0/amp-sidebar-0.1.js
 113.62 kB   |     9.67 kB   |      3.9 kB   |   v0/amp-slides-0.1.js
-123.27 kB   |    12.87 kB   |     4.84 kB   |   v0/amp-social-share-0.1.js
+123.67 kB   |     13.1 kB   |        5 kB   |   v0/amp-social-share-0.1.js
   87.1 kB   |     6.21 kB   |     2.63 kB   |   v0/amp-soundcloud-0.1.js
- 94.93 kB   |     7.83 kB   |      3.1 kB   |   v0/amp-springboard-player-0.1.js
-109.17 kB   |     7.96 kB   |     3.22 kB   |   v0/amp-sticky-ad-0.1.js
-167.23 kB   |     15.6 kB   |     6.38 kB   |   v0/amp-twitter-0.1.js
-125.37 kB   |    10.05 kB   |     3.95 kB   |   v0/amp-user-notification-0.1.js
+ 94.93 kB   |     7.83 kB   |     3.11 kB   |   v0/amp-springboard-player-0.1.js
+109.17 kB   |     7.96 kB   |     3.23 kB   |   v0/amp-sticky-ad-0.1.js
+ 167.5 kB   |    15.67 kB   |     6.41 kB   |   v0/amp-twitter-0.1.js
+125.52 kB   |    10.05 kB   |     3.95 kB   |   v0/amp-user-notification-0.1.js
  86.67 kB   |     6.08 kB   |      2.6 kB   |   v0/amp-vimeo-0.1.js
  86.27 kB   |     5.94 kB   |     2.56 kB   |   v0/amp-vine-0.1.js
-610.64 kB   |   466.47 kB   |   152.72 kB   |   v0/amp-viz-vega-0.1.js
-135.19 kB   |     8.65 kB   |     3.59 kB   |   v0/amp-youtube-0.1.js
-188.64 kB   |    40.39 kB   |    13.96 kB   |   current-min/f.js / current/integration.js
\ No newline at end of file
+610.65 kB   |   466.47 kB   |   152.73 kB   |   v0/amp-viz-vega-0.1.js
+135.34 kB   |     8.65 kB   |     3.59 kB   |   v0/amp-youtube-0.1.js
+191.84 kB   |    40.74 kB   |    14.08 kB   |   current-min/f.js / current/integration.js
```

Comment:
@erwinmombay I ran `gulp build && gulp dist` and then `gulp size` to get the new size. Please confirm the size is acceptable by importing mode.js to integration.js.

Will revert the change to the size file before commit.